packages/stream-model-handler/src/interfaces-utils.ts

```diff
@@ -0,0 +1,641 @@
+import type { Context } from '@ceramicnetwork/common'
```

Comment:
This looks like a lot of code to do schema comparison. It seems like we would do better to store interfaces as essentially a meta schema, and just use schema validation.

Reply:
Interfaces mostly work like regular models, so storing them as meta-schemas would likely require significant changes to multiple parts of the stack that would increase the scope of this project.